<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SMFlangeTaskPanel</class>
 <widget class="QDialog" name="SMFlangeTaskPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>298</width>
    <height>450</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Flange Parameters</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <property name="topMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="AddRemove">
           <property name="text">
            <string>Select</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTreeWidget" name="tree">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="headerHidden">
          <bool>true</bool>
         </property>
         <column>
          <property name="text">
           <string>Name</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>SubElement</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Bend</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Type</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="BendType">
            <item>
             <property name="text">
              <string>Material Outside</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Material Inside</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Thickness Outside</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Offset</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Offset</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="Gui::QuantitySpinBox" name="Offset" native="true">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="unit" stdset="0">
             <string notr="true">mm</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Radius</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="Gui::QuantitySpinBox" name="Radius" native="true">
            <property name="unit" stdset="0">
             <string notr="true">mm</string>
            </property>
            <property name="minimum" stdset="0">
             <double>0.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_5">
            <property name="sizePolicy">
             <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Angle</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="Gui::QuantitySpinBox" name="Angle" native="true">
            <property name="unit" stdset="0">
             <string notr="true">deg</string>
            </property>
            <property name="minimum" stdset="0">
             <double>-360.000000000000000</double>
            </property>
            <property name="maximum" stdset="0">
             <double>360.000000000000000</double>
            </property>
            <property name="value" stdset="0">
             <double>90.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Length</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="Gui::QuantitySpinBox" name="Length" native="true">
            <property name="unit" stdset="0">
             <string notr="true">mm</string>
            </property>
            <property name="minimum" stdset="0">
             <double>0.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="5" column="0" colspan="2">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <property name="topMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>Length mode</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="LengthSpec">
              <item>
               <property name="text">
                <string>Leg</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Outer Sharp</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Inner Sharp</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Tangential</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item row="6" column="0" colspan="2">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QCheckBox" name="UnfoldCheckbox">
              <property name="text">
               <string>Unfold</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="ReversedCheckbox">
              <property name="text">
               <string>Reversed</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Offsets</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Side Offsets</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Gap A</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="Gui::QuantitySpinBox" name="gap1" native="true">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="unit" stdset="0">
             <string notr="true">mm</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Gap B</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="Gui::QuantitySpinBox" name="gap2" native="true">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="unit" stdset="0">
             <string notr="true">mm</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Extend A</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="Gui::QuantitySpinBox" name="extend1" native="true">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="unit" stdset="0">
             <string notr="true">mm</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Extend B</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="Gui::QuantitySpinBox" name="extend2" native="true">
            <property name="unit" stdset="0">
             <string notr="true">mm</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Relief Cuts</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="3" column="0">
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>Width</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_12">
            <property name="sizePolicy">
             <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Depth</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="Gui::QuantitySpinBox" name="reliefWidth" native="true">
            <property name="unit" stdset="0">
             <string notr="true">mm</string>
            </property>
            <property name="minimum" stdset="0">
             <double>0.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="Gui::QuantitySpinBox" name="reliefDepth" native="true">
            <property name="unit" stdset="0">
             <string>mm</string>
            </property>
            <property name="minimum" stdset="0">
             <double>0.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="1" column="0" rowspan="2" colspan="2">
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <property name="topMargin">
             <number>1</number>
            </property>
            <item>
             <widget class="QRadioButton" name="reliefRectangle">
              <property name="text">
               <string>Rectangle</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">reliefTypeButtonGroup</string>
              </attribute>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="reliefRound">
              <property name="text">
               <string>Round</string>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">reliefTypeButtonGroup</string>
              </attribute>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>177</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Miter</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QCheckBox" name="autoMiterCheckbox">
         <property name="text">
          <string>Auto Miter</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupAutoMiter">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Auto Miter Parameters</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <widget class="QLabel" name="label_16">
            <property name="text">
             <string>Minimum Gap</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="Gui::QuantitySpinBox" name="minGap" native="true">
            <property name="unit" stdset="0">
             <string notr="true">mm</string>
            </property>
            <property name="minimum" stdset="0">
             <double>0.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_15">
            <property name="text">
             <string>Max Extend Distance</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="Gui::QuantitySpinBox" name="maxExDist" native="true">
            <property name="unit" stdset="0">
             <string notr="true">mm</string>
            </property>
            <property name="minimum" stdset="0">
             <double>0.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupManualMiter">
         <property name="title">
          <string>Manual Miter</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="2" column="1">
           <widget class="Gui::QuantitySpinBox" name="miterAngle2" native="true">
            <property name="unit" stdset="0">
             <string notr="true">deg</string>
            </property>
            <property name="minimum" stdset="0">
             <double>-360.000000000000000</double>
            </property>
            <property name="maximum" stdset="0">
             <double>360.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="Gui::QuantitySpinBox" name="miterAngle1" native="true">
            <property name="unit" stdset="0">
             <string notr="true">deg</string>
            </property>
            <property name="minimum" stdset="0">
             <double>-360.000000000000000</double>
            </property>
            <property name="maximum" stdset="0">
             <double>360.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>Angle 2</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_14">
            <property name="text">
             <string>Angle 1</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>Perforation</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QCheckBox" name="checkPerforate">
         <property name="text">
          <string>Perforate</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupPerforate">
         <property name="title">
          <string>Perforation Parameters</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="0" column="0">
           <widget class="QLabel" name="label_17">
            <property name="text">
             <string>Angle</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="Gui::QuantitySpinBox" name="perforateAngle" native="true">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="unit" stdset="0">
             <string notr="true">mm</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_18">
            <property name="text">
             <string>Initial Cut Length</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="Gui::QuantitySpinBox" name="perforateInitialCutLen" native="true">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="unit" stdset="0">
             <string notr="true">mm</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_19">
            <property name="text">
             <string>Max Cut Length</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="Gui::QuantitySpinBox" name="perforateMaxCutLen" native="true">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="unit" stdset="0">
             <string notr="true">mm</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_20">
            <property name="text">
             <string>Max Tab Length</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="Gui::QuantitySpinBox" name="perforateMaxTabLen" native="true">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="unit" stdset="0">
             <string notr="true">mm</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>253</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Gui::QuantitySpinBox</class>
   <extends>QWidget</extends>
   <header>Gui/QuantitySpinBox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="reliefTypeButtonGroup"/>
 </buttongroups>
</ui>
